.github/workflows/build-test.yml

```diff
@@ -32,7 +32,10 @@ jobs:
     - name: Install dependencies
       run: |
         pip install .[docs]
-        pip install pytest black isort pytest-cov
```

Comment:
we also need a comment here, and on the pre-commit config file that these two versions should be upgraded together so that we don't forget and they are kept in sync.